src/utils/api.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+import { CookieConsent } from '../script/stores/complianceStore';
+import { tryLocalStorageGet, tryLocalStorageSet } from './local-storage';
+
+const browserInfoKey = 'browserInfo';
```

Comment:
I'm thinking we can simplify this vs the website scenarios this was based on.

We remember the user's ultimate decision to use this site or the nextgen one.
So I think that means it's not worth the code to cache in this layer at all.

Reply:
That makes sense, I have simplified it so that we directly use `fetchBrowserInfo`